File: Python Language/Q27.py
# Program to convert time into total seconds

hours = int(input("Enter Hours: "))
minutes = int(input("Enter Minutes: "))
seconds = int(input("Enter Seconds: "))

# Calculation: 
# 1 hour = 3600 seconds
# 1 minute = 60 seconds
total_seconds = (hours * 3600) + (minutes * 60) + seconds

print(f"Total time in seconds: {total_seconds}")

File: C++ Language/Q27.cpp
#include <iostream>
using namespace std;

int main() {
    int hours, minutes, seconds;
    int totalSeconds;

    cout << "Enter hours: ";
    cin >> hours;
    cout << "Enter minutes: ";
    cin >> minutes;
    cout << "Enter seconds: ";
    cin >> seconds;

    totalSeconds = (hours * 3600) + (minutes * 60) + seconds;

    cout << "Total time in seconds: " << totalSeconds << endl;

    return 0;
}